pkgs/by-name/ai/airscan/package.nix

```
@@ -0,0 +1,25 @@
+{ buildGoModule
+, fetchFromGitHub
+, lib
+}:
+
+buildGoModule rec {
+  pname = "airscan";
+  version = "0.3";
+
+  src = fetchFromGitHub {
+    owner = "stapelberg";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-gk0soDzrsFBh+SrWcfO/quW6JweX6MthOigTHcaq1oE=";
+  };
+
+  vendorHash = "sha256-I5JRGaff6OIwx4q7BjpFwvJiQe4kw03V8+McYPcJhho=";
+
+  meta = with lib; {
+    description = "Package to scan paper documents using the Apple AirScan (eSCL) protocol";
+    homepage = "https://github.com/stapelberg/airscan";
```

The changelog entry could be picked-up by tools (e.g., `nix-update`) to add a link to the release notes to the commit messages.

```suggestion
    homepage = "https://github.com/stapelberg/airscan";
    changelog = "https://github.com/stapelberg/airscan/releases/tag/v${version}";
```